### PR TITLE
Fixed all backup related issues

### PR DESCRIPTION
**Pull request by tboerger**

### CODE REVIEW

File: lib/crowbar/client/command/backup/upload.rb [hunk 1]
```diff
@@ -27,7 +27,8 @@ def request
                 args.file
               else
                 File.new(
-                  args.file
+                  args.file,
+                  File::RDONLY | File::RDWR
```
Comment (vuntz) [inline lib/crowbar/client/command/backup/upload.rb]: Why RDWR?

File: lib/crowbar/client/command/batch/build.rb [hunk 2]
```diff
@@ -35,7 +35,8 @@ def request
                 args.file
               else
                 File.new(
-                  args.file
+                  args.file,
+                  File::RDONLY | File::RDWR
```
Comment (vuntz) [inline lib/crowbar/client/command/batch/build.rb]: Same question here.

File: lib/crowbar/client/command/backup/upload.rb (hunk 1 above)
Comment (tboerger) [inline lib/crowbar/client/command/backup/upload.rb]: Just stupid, fixed.

File: lib/crowbar/client/command/batch/build.rb (hunk 2 above)
Comment (tboerger) [inline lib/crowbar/client/command/batch/build.rb]: Just stupid, fixed.

**Comment by vuntz** [comment]
+1

**Comment by rsalevsky** [comment]
+1

**Pull request merged**